.github/actions/nr-report-build-size/action.yml

```diff
@@ -0,0 +1,47 @@
+# This composite action is used to report the local build size to New Relic
+# You must build the agent before running this action
+
+name: 'Report Build Size to New Relic'
+
+inputs:
+  nr-api-key:
+    description: 'NR API Key for reporting events'
+    required: true
+
+runs:
+  using: "composite"
+  steps:
+    - name: Install dependencies
+      run: npm install --silent --no-progress --prefix $GITHUB_ACTION_PATH/..
+      shell: bash
+    - name: Get local stats
+      id: get-stats
+      shell: bash
+      run: |
+        rum=$(cat ./build/nr-rum-standard.stats.json); echo "rum=$rum" >> $GITHUB_OUTPUT;
+        full=$(cat ./build/nr-full-standard.stats.json); echo "full=$full" >> $GITHUB_OUTPUT;
+        spa=$(cat ./build/nr-spa-standard.stats.json); echo "spa=$spa" >> $GITHUB_OUTPUT;
+    - name: Report rum size to NR
+      uses: metal-messiah/webpack-build-size-action@main
+      with:
+        nr-api-key: ${{ inputs.nr-api-key }}
+        nr-account-id: '550352'
+        nr-env: 'staging'
+        analysis-file-contents: ${{ steps.get-stats.outputs.rum }}
+        file-name-filter: '.min.js'
+    - name: Report full size to NR
+      uses: metal-messiah/webpack-build-size-action@main
+      with:
+        nr-api-key: ${{ inputs.nr-api-key }}
+        nr-account-id: '550352'
```

Comment:
Maybe consider moving values for `nr-account-id` and/or `guid` to GHA variables.

Reply:
will do in next update